File: U4/Java/U4P_6.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompan.u4p_6;
import java.util.Arrays;
/**
 *
 * @author alfre
 */
public class U4P_6 {

    public static void main(String[] args) {
        int[] numeros = { 5, 2, 8, 1, 3 };
        ordenarArreglo(numeros);
        System.out.println("Arreglo ordenado de menor a mayor:");
        imprimirArreglo(numeros);
    }

    static void ordenarArreglo(int[] arreglo) {
        int[] arregloAuxiliar = Arrays.copyOf(arreglo, arreglo.length);
        Arrays.sort(arregloAuxiliar);
        System.arraycopy(arregloAuxiliar, 0, arreglo, 0, arreglo.length);
    }

    static void imprimirArreglo(int[] arreglo) {
        for (int numero : arreglo) {
            System.out.print(numero + " ");
        }
        System.out.println();
    }
}

File: U4/C#/U4P_6.cs
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class U4P_6
    {
        static void Main(string[] args)
        {
            int[] numeros = { 5, 2, 8, 1, 3 };
            OrdenarArreglo(numeros);
            Console.WriteLine("Arreglo ordenado de menor a mayor:");
            ImprimirArreglo(numeros);
        }

        static void OrdenarArreglo(int[] arreglo)
        {
            int[] arregloAuxiliar = new int[arreglo.Length];
            Array.Copy(arreglo, arregloAuxiliar, arreglo.Length);
            Array.Sort(arregloAuxiliar);
            Array.Copy(arregloAuxiliar, arreglo, arreglo.Length);
        }

        static void ImprimirArreglo(int[] arreglo)
        {
            foreach (int numero in arreglo)
            {
                Console.Write(numero + " ");
            }
            Console.WriteLine();
        }
    }
}